menu.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <section class="headline">
+   <div class="works">
+      <h1>My Recent Works</h1>
+      <img src="images/icons/Indicator.svg" alt="Indicator"><br>
+    <div class="work-list">
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <div class="name">Multi-Post Stories Gain+Glory</div>
```

Comment:
- [ ] This is a project title and following the best semantics it should be in a hx. I'll suggest you wrap this in a `h2 or h3` tag. :+1: :100: